### PR TITLE
Cleanup Vehicles (Lombokify Part 2)

### PR DESCRIPTION
**Pull request by strubium**
## 📝 Description

Cleans up more of the codebase, mostly vehicles 

## 🎯 Goals

- Use lombok where possible
- Remove empty/unused files

## ❌ Non Goals

- Make Vehicles functional 

## 🚦 Testing 

Yes

## ⏮️ Backwards Compatibility 

1.12.2

## 📚 Related Issues & Documents

#524 

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📑 Documentation
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 No documentation needed

### CODE REVIEW

File: src/main/java/com/paneedah/weaponlib/HighIQSpawnEgg.java [hunk 1]
```diff
@@ -87,14 +91,10 @@ public Builder withID(int i) {
 
     }
 
-    private String entitySpawnName;
-    private Predicate<Block> blockPredicate;
+    @Setter @Getter private String entitySpawnName;
+    @Setter @Getter private Predicate<Block> blockPredicate;
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/HighIQSpawnEgg.java]: Getter then Setter

File: src/main/java/com/paneedah/weaponlib/LightExposure.java [hunk 2]
```diff
@@ -2,18 +2,20 @@
...
 import net.minecraft.entity.Entity;
 import net.minecraft.world.World;
 
+@NoArgsConstructor
 public class LightExposure extends UniversalObject implements Exposure {
 
     private long lastExposureTimestamp;
     private long maxDuration;
-    private float totalDose = 1f;
+    @Setter @Getter private float totalDose = 1f;
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/LightExposure.java]: Getter then Setter

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

**Review by strubium — COMMENTED** [review]
personally i think its perfect

**Comment by Desoroxxx** [comment]
> personally i think its perfect

obama

**Review by Desoroxxx — APPROVED** [review]

**Pull request merged**